client-src/elements/chromedash-form-field.ts

```diff
@@ -465,12 +481,12 @@ export class ChromedashFormField extends LitElement {
  * if the field is stage-specific, and returns the corresponding value.
  * Returns null if not defined or not found.
  * Handles special cases like shipping milestones and mapped stage fields.
- * @param {string} fieldName
- * @param {number|Object|undefined} stageOrId
- * @param {Array<{name:string, value:*}>} formFieldValues
- * @return {*}
  */
-function getFieldValueWithStage(fieldName, stageOrId, formFieldValues) {
+function getFieldValueWithStage(
+  fieldName: string,
+  stageOrId: number | StageDict | undefined,
+  formFieldValues: FormFieldValue[] & {feature?: Feature}
+): any {
```

Comment:
I'd leave off the explicit `:any` return type so that if and when the contained functions get more precise return types, this one will automatically propagate them.
```suggestion
) {
```

Reply:
Ping.